src/react/viewport-controller.js

```diff
@@ -90,7 +90,7 @@ const propTypes = {
 
 const getDefaultCursor = ({isDragging}) => isDragging ? CURSOR.GRABBING : CURSOR.GRAB;
 
-const defaultProps = {
+const defaultProps = Object.assign({}, DEFAULT_TRANSITION_PROPS, {
```

Comment:
Would be `const defaultProps = Object.assign({}, TransitionManager.defaultProps, {`